pulp_container/app/tasks/sync_stages.py

```diff
@@ -343,7 +343,9 @@ async def get_paginated_tag_list(self, rel_link, repo_name):
         while True:
             link = urljoin(self.remote.url, rel_link)
             list_downloader = self.remote.get_downloader(url=link)
-            await list_downloader.run(extra_data={"repo_name": repo_name})
+            # FIXME this can be rolledback after https://github.com/pulp/pulp_container/issues/1288
+            # tags/list endpoint does not like any unnecessary headers to be sent
+            await list_downloader.run(extra_data={"repo_name": repo_name, "headers": {}})
```

Comment:
FYI the proper solution would be to write a CustomRenderer, https://www.django-rest-framework.org/api-guide/renderers/#custom-renderers  like we have for Blobs and Manifests https://github.com/pulp/pulp_container/blob/main/pulp_container/app/registry_api.py#L89 and add it to every view/viewset